styles/elements/_list.scss

```diff
@@ -223,8 +224,6 @@ Markup:
 Styleguide #{$sgi-listsWithIcons}
 */
 
-$iconfont-size: (16, 18, 22, 30, 100); //Duplicated in _icons.scss
```

Comment:
I'm not sure that if we delete this line, iconfont sizes works from _vars partial